### DISCUSSION TITLE
Keyviz won't open.... or close

### DISCUSSION THREAD

**Discussion by GalinMcMahon97005** [mulaRahul/keyviz#291] [Q&A]
Just installed it and it's fantastic!  However, attempting to open it and it just flashes and closes immediately.  On Win 11.  Suggestions?

**Comment by H-Sapien** [comment]
Yes, I had the same until I noticed that the settings controls were opening on my second screen.
Windows has changed my 1 and 2 screens about lately and I didn't notice. My smaller VGA screen used to be 1, but now it's 2. If you have two screens, even inactive/off ones, Windows may think it's a valid piece of hardware and Keyviz may be obeying what Windows is telling it. 
ATM, Keyviz is popping up on 2; lucky I caught it. YMMV.
...and then it's putting white stripes across the top & bottom of 1, but that's another story.
